src/Nancy.Tests/Unit/Security/ModuleSecurityFixture.cs

```diff
@@ -286,84 +207,15 @@ public void Should_return_null_with_RequiresAnyClaim_and_any_claim_met()
         }
 
         [Fact]
-        public void Should_return_forbidden_response_with_RequiresValidatedClaims_enabled_but_claims_missing()

```

Comment:
Any tests around `RequiresValidatedClaims` were removed. HasClaim(s) performs that function.